src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -131,6 +131,8 @@ public function testGetSetHelp()
         $ret = $command->setHelp('help1');
         $this->assertEquals($command, $ret, '->setHelp() implements a fluent interface');
         $this->assertEquals('help1', $command->getHelp(), '->setHelp() sets the help');
+        $command->setHelp(null);
```

Comment:
setHelp and getHelp both except/return a string. So setting null is not allowed. Btw, the properties in the command should be initialized as empty string. otherwise the phpdoc is also wrong before calling the setters.